### PR TITLE
[For Review] Standard operating procedures section

### PR DESCRIPTION
**Pull request by cgosingtian**
Starting this PR to create the header document for our Git section.

In the table of contents we’ll have Standard Operating Procedures as a main header, with Workflow (and its children) and our Git section as sub headers.

### CODE REVIEW

**Comment by cgosingtian** [comment]
Some changes to workflow.md, README.md here too.

**Comment by kbacalso** [comment]
There seems to be some formatting problems at the numbered list in the Summary section:
screen shot 2016-05-12 at 4 51 59 pm

**Comment by cgosingtian** [comment]
This PR is WIP again, will be updating this based on our last meeting.

File: README.md [hunk 1]
```diff
@@ -1,57 +1,34 @@
...
+
+//Some text here
 
 * [PROJECT ORGANIZATION](/project-organization.md#project-organization)
   * [FINDER FOLDER STRUCTURE](/project-organization.md#finder-folder-structure)
   * [PROJECT NAVIGATOR](/project-organization.md#project-navigator)
-  * [DEPENDENCY MANAGEMENT WITH COCOAPODS](/project-organization.md#dependency-management-with-cocoapods)
-* [REDMINE](/redmine.md#redmine)
-* [BUILD CONFIGURATIONS](/build-configurations.md#build-configurations)
-  * [BUILD SETTINGS](/build-configurations.md#build-settings)
```
Comment (kbacalso) [inline README.md]: Build Settings under Build Configuration doesn't seem right.. I think the two should be on the same level?

Comment (cgosingtian) [inline README.md]: What do you think of just removing Build Settings from the list? Configuration and Settings seem synonymous already.

**Comment by cgosingtian** [comment]
The bug in the display of the Summary section has been fixed, by the way @kbacalso

File: README.md (hunk 1 above)
Comment (kbacalso) [inline README.md]: checked 👍

**Pull request merged**